website/pages/docs/deployment/ecs.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Running CloudQuery on Amazon ECS
+tag: tutorial
+date: 2023/03/03
+---
+
+# Deploy with Amazon ECS
+
+In this tutorial, we will show you how to load AWS resources into Amazon S3 by running CloudQuery as a service in [Amazon ECS](https://github.com/features/actions), using the AWS source- and S3 destination plugins.
```

Comment:
I think a little architecture diagram of how the components fit together would be nice here, so you 1) can see what you're getting if you're using the template, or at least 2) see what a good deployment strategy looks like in general